drivers/include/mtd_sdcard.h

```diff
@@ -48,18 +49,37 @@ typedef struct {
  * @ingroup config_drivers_storage
  * @{
  */
+/**
+ * @brief   Enable SDCard Erase
+ * @note    SDCards handle sector erase internally so it's
+ *          possible to directly write to the card without erasing
+ *          the sector first.
+ *          Attention: an erase call will therefore NOT touch the content,
+ *          so enable this feature to ensure overriding the data.
+ */
+#ifdef DOXYGEN
+#define CONFIG_MTD_SDCARD_ERASE
+#endif
+/** @} */
+
 /**
  * @brief   Enable Skip SDCard Erase
  * @note    SDCards handle sector erase internally so it's
- * possible to directly write to the card without erasing
- * the sector first.
- * Attention: an erase call will therefore NOT touch the content,
- * so disable this feature to ensure overriding the data.
+ *          possible to directly write to the card without erasing
+ *          the sector first.
+ *          Attention: an erase call will therefore NOT touch the content,
+ *          so disable this feature to ensure overriding the data.
+ *
+ *          @deprecated Use inverse @ref CONFIG_MTD_SDCARD_ERASE instead.
+ *          Will be removed after 2021.01 release.
  */
 #ifndef MTD_SDCARD_SKIP_ERASE
+#if IS_ACTIVE(CONFIG_MTD_SDCARD_ERASE)
+#define MTD_SDCARD_SKIP_ERASE (0)
```

Comment:
On an offline chat @MichelRottleuthner indicated that this functionality is actually not supported yet. So I would say it is safe to change the macro logic without deprecating this. Let's just replace the macro

Reply:
Okay. I shall update

Reply:
Done.